src/internal/mod.rs

```diff
@@ -38,7 +38,8 @@ lazy_static! {
         .to_string(),
         _ => "https://api.ironcorelabs.com/api/1/".to_string(),
     };
-    pub static ref OUR_REQUEST: IronCoreRequest = IronCoreRequest::new(URL_STRING.as_str());
+    // pub static ref OUR_REQUEST: IronCoreRequest =
```

Comment:
I wonder if it would be appropriate to open an issue on reqwest to better understand why this doesn't work?